chapter/evaluation.md

```diff
@@ -37,10 +37,189 @@ In particular, it details the survey ([@sec:qualitative]) which was conducted wi
 
 ### Quantitative
 
-## Process
 
 ## Results
 
+### Process
 ### Qualitative
 
+
+As outlined in [#sec:qualitative-study-outline], the qualitative study consists of two parts conducted before and after an introductory workshop.
+The pre-evaluation aimed to catch the users's expected features and behaviours, while the main survey asked users about their concrete experiences with the NLS.
+
+#### Pre-Evaluation
+
+Responding to the first point (c.f. [#sec:expected-features]), the participants unanimously identified four of the six foundational language server capabilities that guided the implementation of the project (c.f. [@sec:commands-and-notifications, @langserverorg] );
+
+Type-information on hover
+  ~ was named almost uniformly.
+    The participants showed a special interest in this feature describing specific behaviours.
+    The desired information exposed by this feature are value types including applied contracts and documentation as well as function signatures.
```

Comment:
```suggestion
    The desired information exposed by this feature are types (including applied contracts) and documentation as well as function signatures.
```

Are types and function signatures really different?

Reply:
I dont think so but is was mentioned explicitly.
As in: for functions - function signatures and documentation